generator/templates/server/parameter.gotmpl

```diff
@@ -173,7 +173,7 @@ func ({{ .ReceiverName }} *{{ pascalize .Name }}Params) BindRequest(r *http.Requ
         break
       }
     }
-    {{ else }}if err := body.Validate(route.Formats); err != nil {
+    {{ else if or .IsComplexObject .IsTuple }}if err := body.Validate(route.Formats); err != nil {
```

Comment:
As far as I can tell `isInterface` is true with

```yaml
type: object
properties:
  name:
    type: string
```

which is what lead me to using these properties instead.